tests/lint/unittest_lint.py

```diff
@@ -785,6 +785,40 @@ def test_custom_should_analyze_file():
         assert "invalid syntax" in messages[0]
 
 
+# we do the check with jobs=1 as well, so that we are sure that the duplicates
+# are created by the multiprocessing problem.
+@pytest.mark.parametrize("jobs", [1, 2])
+def test_multiprocessing(jobs):
+    """ Check that multiprocessing does not create duplicates.
+    """
+    source_dir = os.path.join(REGRTEST_DATA_DIR)
```

Comment:
Unless I'm mistaken we can use `REGRTEST_DATA_DIR` directly ?

Reply:
Yeah, that was a left-over from the evolution of the test :-)
Fixed.